Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -118,7 +118,11 @@
 {
     m_lastSentSize = m_size;
     m_isWaitingForDidUpdateGeometry = true;
-    m_webPageProxy.send(Messages::DrawingArea::UpdateGeometry(m_size, false /* flushSynchronously */, MachSendRight()), m_identifier);
+    m_webPageProxy.sendWithAsyncReply(Messages::DrawingArea::UpdateGeometry(m_size, false /* flushSynchronously */, MachSendRight()), [weakThis = WeakPtr { this }] {
+        if (!weakThis)
+            return;
+        weakThis->didUpdateGeometry();
```

Comment:
It would be good to Ref this before calling didUpdateGeometry(), something like:
```
if (Ref protectedThis = weakThis.get())
    protectedThis->didUpdateGeometry();
```

Reply:
RefPtr.  Sure.

Reply:
Actually, no, because they're not refcounted.

Reply:
ahah :)